tests/test_definitions/column_map_expectations/expect_column_values_to_be_unique.json

```diff
@@ -155,7 +157,22 @@
           "unexpected_list": ["1", "1", "2", "2"],
           "unexpected_index_list": [0, 1, 2, 3]
         }
-      },
+      }
+    ]
+  },
+   {
+    "data" : {
+      "0": [1, 2, 3, 4],
+      "_c0": [1, 2, 3, 4]
+    },
+    "schemas": {
+      "spark": {
+        "0": "IntegerType",
+        "_c0": "IntegerType"
+      }
+    },
+    "only_for": ["spark"],
+    "tests" : [
       {
```

Comment:
These changes, along with ones to `test_expectations.py` and `test_expectations_cfe.py` were added as part of #4073